kokoro/gcp_ubuntu/bazel/bindings/google.cfg

```diff
@@ -0,0 +1,19 @@
+# Format: //devtools/kokoro/config/proto/build.proto
+
+# Copyright 2019 Google LLC
```

Comment:
lol @ blank files still needing copyright headers...
(also nit: these could be 2020?)

Reply:
Yeah it's a bit silly, but I think it's also not worth making an exception for. Either way it doesn't matter and this not mattering seems slightly easier.

And this highly valuable IP is actually copied from another file, so the copyright year should stay, I think.